automobile/buildings.md

```diff
@@ -88,6 +90,10 @@ The available roof types are:
   - metal tiles
   - bitumen
 
+The available colored roof types are:
+  - tiled
+  - bitumen
```

Comment:
Instead of duplicating the list I would put in parenthesis in the first list `(supports color)`.